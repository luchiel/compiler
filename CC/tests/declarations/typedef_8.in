typedef int d;
typedef struct c {int b;} d;
