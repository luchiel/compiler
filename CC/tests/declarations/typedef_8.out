Error: RedefinedSymbol:2:28 d
